docs/warning-level.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: default
+title: Warning Level
+nav_order: 93
+description: "Labs64 NetLicensing warning level"
+permalink: warning-level
+---
+
+{{ page.title }}
+================
+
+The NetLicensing Warning Level feature provides vendors with valuable metrics regarding the current licensing status of a specific customer. Represented as a traffic light, the Warning Level indicates whether a customer profile is fully licensed, approaching under-licensing, or has reached under-licensing.
+
+Important Clarifications
+------------------------
+
+- **Warning Level** vs. **Valid Parameter**: The Warning Level should not be confused with the `valid` parameter in the validation response. While the valid parameter represents the current validity of a specific licensing model for a given customer, the Warning Level offers an informative status of the licensing situation, highlighting potential or imminent under-licensing issues.
+- **Purpose**: The Warning Level is designed to be an indicator of the licensee's health and is not intended to replace the validation response. It serves solely as a supplementary metric to provide insights into licensing status. 
+
+Statuses
+--------
+
+The Warning Level is calculated during each validation request and is represented by the following statuses:
+-	GREEN – the customer is licensed for all configured product modules
+-	YELLOW – some licensing parameters are approaching under-licensed status
+-	RED – some licensing parameters have reached under-licensed status
+
+**Note:** The warning level status resets after a certain period, usually due to extended licensee inactivity or delayed validation requests. When this occurs, the customer status is shown as `GREY (undefined)`. You can force an update of the Warning Level by sending a validation request with the parameter `dryRun` set to `true`.
```

Comment:
```suggestion
Supported Warning Levels
--------

The Warning Level can have one of the following values:
- **GREEN (Licensed):** The customer is licensed for all configured product modules.
- **YELLOW (Warning):** Some product modules are approaching under-licensed status.
- **RED (Underlicensed):** Some product modules have reached under-licensed status.
- **GREY (Unknown):** The warning level was not calculated or expired since the last update.

**Note:** The warning level is calculated with each validation request from the customer and cached for a period of time. The value may be evicted from cache due to extended inactivity (no validation requests) or license changes since the last validation. In such cases, the warning level is reported as `GREY (Unknown)`. To force an update, send a validation request with the `dryRun` parameter set to `true` (or refresh button in the UI). Be aware that some licenses only activate on the first non-dry validation request, which may cause the warning level to be incorrectly reported as RED or YELLOW if updated via a dry validation.
```